Update regarding adding a DMA to a DMS (#3709)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Functionality/DataMiner_Systems/Adding_a_DMA/Adding_a_regular_DataMiner_Agent.md

> - If the DataMiner System uses [Dedicated clustered storage](xref:Dedicated_clustered_storage), make sure the new DMA is configured to use the same clustered storage as the DMS you are about to add it to (by modifying the general database settings in its [DB.xml](xref:DB_xml) file to match those of the other DMAs and then restarting the DMA).
> - If the DataMiner System uses STaaS, additional steps are required. See [Adding a DataMiner Agent to a DMS running STaaS](xref:Adding_a_DMA_to_a_DMS_running_STaaS).

1. On a DMA that is currently already in the cluster, go to the *System Center* module and select the *Agents* page.

1. In the *Manage* tab, below the list of DMAs in the cluster, click *Add*.

1. Enter the IP address of the DMA you want to add and click *Add*.

> > [!NOTE]
> > You should not set the cluster name for each DMA. This will be done automatically upon adding the DMA to the DMS.
> 
> > [!NOTE]
> > To add a Failover pair to a DataMiner System, first add the primary DMA as a regular DataMiner Agent, and then configure Failover. See [Preparing the two DataMiner Agents](xref:Preparing_the_two_DataMiner_Agents).

Reviewer: ```suggestion
> [!NOTE]
>
> - It is not necessary to set the cluster name when you add a DMA to a DMS. The name will be set automatically.
> - To add a Failover pair to a DataMiner System, first add the primary DMA as a regular DataMiner Agent, and then configure Failover. See [Preparing the two DataMiner Agents](xref:Preparing_the_two_DataMiner_Agents).
```

> - If the DataMiner System uses [Dedicated clustered storage](xref:Dedicated_clustered_storage), make sure the new DMA is configured to use the same clustered storage as the DMS you are about to add it to (by modifying the general database settings in its [DB.xml](xref:DB_xml) file to match those of the other DMAs and then restarting the DMA).
> - If the DataMiner System uses STaaS, additional steps are required. See [Adding a DataMiner Agent to a DMS running STaaS](xref:Adding_a_DMA_to_a_DMS_running_STaaS).

1. On a DMA that is currently already in the cluster, go to the *System Center* module and select the *Agents* page.

1. In the *Manage* tab, below the list of DMAs in the cluster, click *Add*.

1. Enter the IP address of the DMA you want to add and click *Add*.

> [!NOTE]
>

> > - You should not set the cluster name for each DMA. This will be done automatically upon adding the DMA to the DMS.

Author: My change from "It is not necessary to" to "You should not" was done after Bert mentioned to me that "It is not necessary to" still sounds rather optional and you can still set a cluster name if you want to do this.

Yes, this can have a negative effect as this could lead to the wrong agent being interpreted as being "in sync". This will cause data to be lost from the other agent. However, this can only happen if the user doesn't follow the exact procedure. By not setting this this can never happen, thus this is a more robust procedure.
